app/api/domains/api.py

```diff
@@ -126,11 +126,11 @@ async def api_calculate_pp(
     mods: int = Query(0, min=0, max=2_147_483_647),
     mode: int = Query(0, min=0, max=11),
     combo: int = Query(None, max=2_147_483_647),
-    acclist: Optional[str] = Query(None, alias="acc")
+    acclist: list[float] = Query(None, alias="acc")
```

Comment:
@infernalfire72 hey do you remember off the top of your head if this default `None` is correct? i always forget how this works in fastapi

Reply:
I replaced it now anyways, will do another push soon

Reply:
double checked with fastapi docs: https://fastapi.tiangolo.com/tutorial/query-params-str-validations/

u should use `Query([], alias="acc")`

Reply:
That kinda makes me wonder if they pass the same list instance all the way down the stack, in that case it could be susceptible to the same issues as mutable parameter defaults LOL

Reply:
yeah it would be if true, doesn’t matter in our case tho.. we’re not mutating the list, surely?

Reply:
I'll test this later, I need to know now LOL.

If it does behave that way, best to use `default_factory` as to not promote an antipattern.